.github/workflows/self-scheduled-amd.yml

```diff
@@ -438,7 +438,7 @@ jobs:
           CI_SLACK_CHANNEL_DUMMY_TESTS: ${{ secrets.CI_SLACK_CHANNEL_DUMMY_TESTS }}
           CI_SLACK_REPORT_CHANNEL_ID: ${{ secrets.CI_SLACK_CHANNEL_ID_DAILY_AMD }}
           ACCESS_REPO_INFO_TOKEN: ${{ secrets.ACCESS_REPO_INFO_TOKEN }}
-          CI_EVENT: Scheduled CI (AMD)
+          CI_EVENT: Scheduled CI (AMD) - ${{ inputs.gpu_flavor }}
```

Comment:
Just a tiny fix so we can see 

```
Results of the Scheduled CI (AMD) - mi250 tests.
```
instead of 
```
Results of the Scheduled CI (AMD) tests.
```
on the slack